saa_victoria_subtheme.theme

```diff
@@ -29,28 +29,61 @@ function saa_victoria_subtheme_preprocess_page(&$variables) {
  * Implements hook_preprocess_HOOK().
  */
 function saa_victoria_subtheme_preprocess_ds_entity_view(&$variables) {
-  if (isset($variables['content']['#paragraph']) && $variables['content']['#paragraph']->bundle() == 'stanford_card') {
 
-    // set initially from settings
-    // get card variant info - if null, will use the 3rd value
+  if (isset($variables['content']['#paragraph']) && $variables['content']['#paragraph']->bundle() == 'stanford_card') {
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_type', 'su-card-wrapper--icon');
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_border_style', 'su-card-wrapper--border-regular');
-    $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'horizontal_card_image_style');
-    $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'horizontal_card_image_position');
 
-    $variables['content']['#fields']['card_image']['su_card_media'][0]['#stanford_media_image_style'] = 'full_responsive';
-  }
-  // if banner has an image, add a class
+    // Check for media in a card then set classes.
+    if(isset($variables['content']['su_card_media']['#object']) ) {
+      // Set initially from settings
+
+       $variables['card_info'] = [];
+       $variables['card_info']['card_type'] = [];
+       $variables['attributes']['class'][] = $variables['card_info']['card_type'];
+
+       $variables['card_info']['card_type'] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_type', 'su-card-wrapper--icon');
+       $variables['card_info']['horizontal_card_image_style'] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'horizontal_card_image_style');
+
+      // Icon card check
+      if($variables['card_info']['card_type'] == 'su-card-wrapper--icon' )  {
+        $variables['content']['#fields']['card_image']['su_card_media'][0]['#stanford_media_image_style'] = 'full_responsive';
+      }
+
+      // Vertical card check
+      if($variables['card_info']['card_type'] === 'su-card-wrapper--vertical' )  {
+        $variables['content']['#fields']['card_image']['su_card_media'][0]['#stanford_media_image_style'] = 'card_2_1';
+      }
+
+      // All horizontal
+      if($variables['card_info']['card_type'] === 'su-card-wrapper--horizontal' )  {
+
+        // Small circle image
+        if($variables['card_info']['card_type'] === 'su-horizontal-card-wrapper--small-circle' )  {
```

Comment:
This should be
if($variables['card_info']['horizontal_card_image_style'] === 'su-horizontal-card-wrapper--small-circle' )  {

Reply:
Done.